livingatlas/pipelines/src/main/java/au/org/ala/pipelines/parser/UncertaintyParser.java

```diff
@@ -4,13 +4,16 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import lombok.extern.slf4j.Slf4j;
+import org.elasticsearch.common.Strings;
```

Comment:
this is a strange import. i dont think this code should have elasticsearch dependencies.

Reply:
For checking if string is empty or null. Changed to google.common now